File: Python/Kyu 8/Calculate Price Excluding VAT.py
def excluding_vat_price(price, vat=15):
    try:
        return round(price/(1+vat/100.), 2)        
    except:
        return -1


def assert_equals (result, expected):
    if result != expected:
        print(f"Fail. Expected: ${expected}, result: ${result}.")

assert_equals(excluding_vat_price(230.00), 200.00)
assert_equals(excluding_vat_price(123), 106.96)

File: JavaScript/Kyu 8/Calculate price excluding VAT.js
function excludingVatPrice(price){
    return price != null ? Math.round((price/1.15)*100)/100 : -1
}


const excludingVatPrice = (price) => price != null ? Math.round((price/1.15)*100)/100 : -1

// others solutions
excludingVatPrice = p => p === null ? -1 : +(p / 1.15).toFixed(2);



  //---------------------------------------------------------

Test.describe('Fixed Tests', _ => {
    Test.it("Test", __ => {
        Test.assertSimilar(excludingVatPrice(230), 200.00);
        Test.assertSimilar(excludingVatPrice(123), 106.96);
    });
});